src/modules/service/VoteraService.ts

```diff
@@ -0,0 +1,172 @@
+/*******************************************************************************
+
+    This file contains the implementation of data collection service from votera.
+
+    Copyright:
+        Copyright (c) 2020-2021 BOSAGORA Foundation
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+import * as cron from "node-cron";
+import Stoa from "../../Stoa";
+import { VoteraClient } from "../votera/VoteraClient";
+import { URL } from "url";
+import { IMetaData, IPendingProposal } from "../../Types";
+import { logger } from "../common/Logger";
+import { Operation } from "../common/LogOperation";
+import { HeightManager } from "../common/HeightManager";
+
+export class VoteraService {
+
+    /**
+     * The Votera endpoint
+     */
+    public votera: VoteraClient;
+
+    /**
+      * Job to collect Unprocessed Proposals data.
+      */
+    private job: cron.ScheduledTask | null = null;
+
+    /**
+     * Job execution status
+     */
+    private status: boolean;
+
+    constructor(votera_endpoint: URL) {
+        this.votera = new VoteraClient(votera_endpoint)
+        this.status = false;
+    }
+
+    /**
+     * @param StoaInstance
+     * Asynchronously start Proposal Data sync service
+     */
```

Comment:
This requires a document for the return value.
Please add a more detailed explanation.